external-adapter/src/requester.ts

```diff
@@ -1,24 +1,31 @@
-const axios = require('axios')
-const { AdapterError } = require('./errors')
-const { logger } = require('./logger')
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios'
+import { AdapterError } from './errors'
+import { logger } from './logger'
+import { Config } from '@chainlink/types'
+import { util } from '@chainlink/ea-bootstrap'
 
-class Requester {
-  static async request(config, customError, retries = 3, delay = 1000) {
+const defaultCustomError = () => false
```

Comment:
IMHO we hide a simple fn behind an obscure name and reading through the code makes it harder to understand what's exactly going on. I would just call it:

```suggestion
const getFalse = () => false
```